swagger_server/utils/db_utils.py

```diff
@@ -1,67 +1,51 @@
 import logging
 import os
 
-import _pickle as pickle
-import dataset
+import pymongo
 
+DB_NAME = os.environ.get("DB_NAME")
 DB_CONFIG_TABLE_NAME = os.environ.get("DB_CONFIG_TABLE_NAME")
+MONGODB_CONNSTRING = os.environ.get("MONGODB_CONNSTRING")
 
 
 class DbUtils(object):
     def __init__(self):
-        # self.table_name
-        self.config_table = DB_CONFIG_TABLE_NAME
+        self.db_name = DB_NAME
+        self.config_table_name = DB_CONFIG_TABLE_NAME
+        self.mongo_client = pymongo.MongoClient(MONGODB_CONNSTRING)
```

Comment:
Yeah, both ways should work the same, I think we can stick with the MONGODB_CONNSTRING for now. Because some fields, such as MONGO_PORT may not exist , so the format will break.

Reply:
But maybe we can do that in another round of refactoring.